src/Helpers.ts

```diff
@@ -0,0 +1,25 @@
+/** function latinCheck*/
+// no cyrillic letters allowed
+function latinCheck(string: string): boolean {
```

Comment:
Функции лучше называть с глаголов. 
```suggestion
function containsCyrillic(string: string): boolean {
```